config/configuration.go

```diff
@@ -15,6 +15,7 @@ const (
 	SocketPrivateKeyFile     string = "SocketPrivateKeyFile"
 	SocketCertificateFile    string = "SocketCertificateFile"
 	SocketCAFile             string = "SocketCAFile"
+	SocketTLSForTesting      string = "SocketTLSForTesting"
```

Comment:
The naming, was really tough one.
`SocketTLSInsecureSkipVerify` was one of candidates.

With that name `SocketTLSInsecureSkipVerify `, I think that the flag should evenly affect with the certs settings since it's a transparent option to the Go's tls library config.
(In the current patch I made `SocketTLSForTesting` being weaker than certs settings)